extensions/schema-compare/src/controllers/mainController.ts

```diff
@@ -32,7 +32,7 @@ export default class MainController implements vscode.Disposable {
 	}
 
 	private initializeSchemaCompareDialog(): void {
-		azdata.tasks.registerTask('schemaCompare.start', (profile: azdata.IConnectionProfile) => new SchemaCompareDialog().openDialog(profile));
+		azdata.tasks.registerTask('schemaCompare.start', (profile: azdata.IConnectionProfile) => new SchemaCompareResult().start(profile));
```

Comment:
I am temped to rename SchemaCompareResult to SschemaComparePage/Editor since its not result anymore :) But I understand that will be an intrusive/big change...

Reply:
yup I was thinking that too. Probably should be done in a separate PR :)